### PR TITLE
Update che4z-release-information.adoc

### PR DESCRIPTION
**Pull request by DStatWriter**
Updated and Expanded Che4z Release info

Signed-off-by: Daniel Statham <Daniel.Statham@Broadcom.com>

### What does this PR do?
Expands and updates Che4z Release Notes

### What issues does this PR fix or reference?
#15288

### CODE REVIEW

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 1]
```diff
@@ -1,33 +1,41 @@
----
+----
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
---
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 2]
```diff
@@ -1,33 +1,41 @@
...
 title: Eclipse Che4z Release Information
 keywords: 
 tags: [Che4z]
 sidebar: che_7_docs
 permalink: che-7/che4z-release-information/
 folder: che-7/extensions
-summary: 
----
+summary:
+----
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
---
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 3]
```diff
@@ -1,33 +1,41 @@
...
 
 [id="che4z-release-information"]
-= Che4z release information
+= Che4z Release Information
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]: We don't use title case for headings in Che docs.
```suggestion
= Che4z release information
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 4]
```diff
@@ -1,33 +1,41 @@
...
+* COBOL Language Support
+* Explorer for Endevor
+* Zowe CLI and Zowe Explorer (Including Plugins)
 
-=== Version 0.81
+=== Release notes - COBOL Language Support
+
+==== Version 0.8.3
+
+* Java automatic check fixed to recognize all supported JRE/JDK java implementations.
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
* Java automatic check fixed to recognize all supported JRE and JDK implementations.
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 5]
```diff
@@ -1,33 +1,41 @@
...
-Eclipse Che4z currently includes one product, COBOL Language Support.
+Eclipse Che4z currently includes the following products:
 
-== Release notes - COBOL Language Support
+* COBOL Language Support
+* Explorer for Endevor
+* Zowe CLI and Zowe Explorer (Including Plugins)
 
-=== Version 0.81
+=== Release notes - COBOL Language Support
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
== Release notes - COBOL Language Support
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 6]
```diff
@@ -1,33 +1,41 @@
...
+
+==== Version 0.8.3
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
=== Version 0.8.3
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 7]
```diff
@@ -1,33 +1,41 @@
...
+
+* Java automatic check fixed to recognize all supported JRE/JDK java implementations.
+
+==== Version 0.81
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
=== Version 0.81
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 8]
```diff
@@ -1,33 +1,41 @@
...
 
 Version 0.8 was released on 1 October 2019.
 
 * Extension name updated.
 * Repository corrected including improved description.
 * Unexpected error message resolved. Re-directed to internal log.
 
-=== Version 0.8
+==== Version 0.8
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
=== Version 0.8
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 9]
```diff
@@ -38,6 +46,36 @@ Checks for mistakes and errors in COBOL code.
 * *Third-party COBOL syntax highlighting support* +
 Enables syntax highlighting for COBOL code as long as you have an appropriate third-party syntax highlighting extension installed.
 
-== Third-party software agreements
+=== Release notes - Explorer for Endevor
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
== Release notes - Explorer for Endevor
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 10]
```diff
@@ -38,6 +46,36 @@ Checks for mistakes and errors in COBOL code.
...
+
+==== Version 0.8.0
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
=== Version 0.8.0

```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 11]
```diff
@@ -38,6 +46,36 @@ Checks for mistakes and errors in COBOL code.
...
+Version 0.8 was released on 1 August 2019 and includes the following features:
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc:52]:
```suggestion
Version 0.8 was released on 1 August 2019 and included the following features:
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 12]
```diff
@@ -38,6 +46,36 @@ Checks for mistakes and errors in COBOL code.
...
+* Explore Endevor Data Maps
+- Explorer for Endevor lets you browse through Endevor data without any prior knowledge of the file paths being necessary
+
+* Create Filters Based on Your Searches
+- Explorer for Endevor lets you create customer filters based on the searches you perform in the Map View
+
+* Customisable Filters
+- Explorer for Endevor lets you create customisable filters for ease of browsing data sets.
+
+==== Version 0.9.0
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
=== Version 0.9.0
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 13]
```diff
@@ -38,6 +46,36 @@ Checks for mistakes and errors in COBOL code.
...
+=== Release notes - Explorer for Endevor
+
+==== Version 0.8.0
+Version 0.8 was released on 1 August 2019 and includes the following features:
+
+* Explore Endevor Data Maps
+- Explorer for Endevor lets you browse through Endevor data without any prior knowledge of the file paths being necessary
+
+* Create Filters Based on Your Searches
+- Explorer for Endevor lets you create customer filters based on the searches you perform in the Map View
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
- Explorer for Endevor lets you create customer filters based on the searches you perform in the Map View.
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 14]
```diff
@@ -38,6 +46,36 @@ Checks for mistakes and errors in COBOL code.
...
 Enables syntax highlighting for COBOL code as long as you have an appropriate third-party syntax highlighting extension installed.
 
-== Third-party software agreements
+=== Release notes - Explorer for Endevor
+
+==== Version 0.8.0
+Version 0.8 was released on 1 August 2019 and includes the following features:
+
+* Explore Endevor Data Maps
+- Explorer for Endevor lets you browse through Endevor data without any prior knowledge of the file paths being necessary
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
- Explorer for Endevor lets you browse through Endevor data without any prior knowledge of the file paths being necessary.
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 15]
```diff
@@ -38,6 +46,36 @@ Checks for mistakes and errors in COBOL code.
...
+
+* Create Filters Based on Your Searches
+- Explorer for Endevor lets you create customer filters based on the searches you perform in the Map View
+
+* Customisable Filters
+- Explorer for Endevor lets you create customisable filters for ease of browsing data sets.
+
+==== Version 0.9.0
+Version 0.9 was released on 19 September 2019 and includes the following features:
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
Version 0.9 was released on 19 September 2019 and included the following features:
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 16]
```diff
@@ -38,6 +46,36 @@ Checks for mistakes and errors in COBOL code.
...
+
+* Advanced Create Filter function
+- Experienced users can create custom filters without having to use Map View
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
- Experienced users can create custom filters without having to use Map View.
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 17]
```diff
@@ -38,6 +46,36 @@ Checks for mistakes and errors in COBOL code.
...
+
+* Improved Host Creation process
+- Streamlined process to create new Host connections
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
- Streamlined process to create new Host connections.
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 18]
```diff
@@ -38,6 +46,36 @@ Checks for mistakes and errors in COBOL code.
...
+
+* UX Enhancements
+
+=== Zowe Explorer and Zowe CLI
+- link:https://docs.zowe.org/stable/[Zowe CLI and Plugin Documentation]
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc:76]:
```suggestion

- link:https://docs.zowe.org/stable/[Zowe CLI and Plugin Documentation]
```

**Review by rkratky — CHANGES_REQUESTED** [review]

**Comment by DStatWriter** [comment]
Requested changes applied as single commit. Please re-review and merge if satisfied.

Thanks, Dan

**Review by rkratky — APPROVED** [review]

**Pull request merged**